### ISSUE TITLE
add a validation data option (validation_data)

### ISSUE THREAD

**Issue by david-thrower** [david-thrower/cerebros-core-algorithm-alpha#99]
Kind of issue:  feature-request-or-enhancement: Add a parameter validation_data: default: "", which is translated to None on self.validation_data

- Need a parameter to forward validation data to the TensorFlow API
    - self.validation_data -> tf.keras.Model.fit(validation_split=**0.0**,
    validation_data**=[self.validation_data]**,
) ) 

Expected behavior

Be able to  use a customized validation set, not the random split Tensorflow uses.
